### PR TITLE
Issue 176

### LINKED ISSUE #176: Migrate all <a href> tags to link_to
Let's be consistent in the application and use `link_to` instead of `<a>` tags.

### PR DESCRIPTION
**Pull request by SamFritz**
This migrates `<a href=` tags to `link_to` and partially resolves #176. 
It affects the following pages:
- About 
- Documentation

# How should this be tested?

Will be reviewing @ianmilligan1 PR for the second half of migration.

### CODE REVIEW

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
 
   <h3 class=about_h3>What is Web Archiving, what are WARCS, and why should I care?</h3>
     <p class="about_p"><strong>Web archiving</strong> is the process of preserving born-digital content on the World Wide Web. Thinking about how curated content is organized, there are two main components:</p>
     <ul class="about_ul">
-      <li class="about_li"><strong>WARCs (Web ARChive)</strong>: the <a href="https://www.iso.org/obp/ui/#iso:std:iso:28500:ed-2:v1:en" target="_blank">ISO Standard (28500:2017)</a> file format that will house one or more WARC records.</li>
+      <li class="about_li"><strong>WARCs (Web ARChive)</strong>: the <%= link_to('AISO Standard (28500:2017)', 'https://www.iso.org/obp/ui/#iso:std:iso:28500:ed-2:v1:en', target: '_blank') %> file format that will house one or more WARC records.</li>
```
Comment (ianmilligan1) [inline app/views/pages/about.html.erb]: `AISO` -> `ISO`

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -36,30 +36,31 @@
...
+    <p class="about_p">This work is primarily supported by the <%= link_to('Andrew W. Mellon Foundation', 'https://mellon.org/', target: '_blank') %>, the <%= link_to('University of Waterloo', 'http://uwaterloo.ca', target: '_blank') %>, and  <%= link_to('York University Libraries', 'https://www.library.yorku.ca/web/', target: '_blank') %>.</p>
+    <p class="about_p">Other financial and in-kind support comes from the <%= link_to('Social Sciences and Humanities Research Council', 'http://www.sshrc-crsh.gc.ca', target: '_blank') %>, <%= link_to('Compute Canada', 'https://www.computecanada.ca', target: '_blank') %>,the <%= link_to('Ontario Ministry of Research, Innovation, and Science', 'https://www.ontario.ca/page/ministry-research-innovation-and-science', target: '_blank') %>, and <%= link_to('Start Smart Labs', 'http://www.startsmartlabs.com', target: '_blank') %>
```
Comment (ianmilligan1) [inline app/views/pages/about.html.erb]: need a space after the comma after Compute Canada

i.e.
`,the`

should be

`, the`

File: app/views/pages/about.html.erb [hunk 3]
```diff
@@ -7,12 +7,12 @@
...
-    <p class="about_p">AUK is a component of the <a href="https://archivesunleashed.org" target="_blank">Archives Unleashed Project</a>, which aims to allow researchers, scholars, librarians and archivists the ability to access and investigate their web archival collections. As accessibility is a main priority of the project, AUK supports this goal by providing a web-based front end for users to access the <a href="https://archivesunleashed.org/aut/" target="_blank">Archives Unleashed Toolkit</a>.</p>
+    <p class="about_p">AUK is a component of the <%= link_to('Archives Unleashed Project', 'https://archivesunleashed.org', target: '_blank') %>, which aims to allow researchers, scholars, librarians and archivists the ability to access and investigate their web archival collections. As accessibility is a main priority of the project, AUK supports this goal by providing a web-based front end for users to access the <%= link_to('Archives Unleashed Toolkit.', 'https://archivesunleashed.org/aut/', target: '_blank') %>
```
Comment (ianmilligan1) [inline app/views/pages/about.html.erb]: Super pedantic: could we move the period after "Archives Unleashed Toolkit" outside of the link? (just for consistency with rest of site)

Comment (SamFritz) [reply app/views/pages/about.html.erb]: Thanks for the review!

**Comment by ianmilligan1** [comment]
Looks good to me! I'll leave the merging honours to you, @ruebot, if it checks out on your end.

**Pull request merged**